src/DataStructures/DataBox/Prefixes.hpp

```diff
@@ -95,4 +95,13 @@ struct NormalDotNumericalFlux : db::DataBoxPrefix {
   using tag = Tag;
   static constexpr db::DataBoxString label = "NormalDotNumericalFlux";
 };
+
+/// \ingroup DataBoxTagsGroup
+/// \brief Prefix indicating the divergence
+template <typename Tag>
```

Comment:
This tag won't work as a compute item, I'm working on a PR for `Tags::deriv` that does this so we will want to update `div` to be the same